ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
```

Comment:
i agree with telling people what features are coming, though maybe not necessary to talk about features that have already come. obv, we are on the same page about more frequent releases.